packages/ketcher-polymer-editor-react/src/components/shared/ui/Tabs/Tabs.tsx

```diff
@@ -29,44 +28,79 @@ function TabPanel({ children, value, index, ...other }) {
   )
 }
 
-function CustomTabs(props) {
+function a11yProps(index: number) {
+  return {
+    id: `simple-tab-${index}`,
+    'aria-controls': `simple-tabpanel-${index}`
+  }
+}
+
+function CustomTabs(props): ReactElement {
   const [tabIndex, setTabIndex] = useState(0)
-  const { tabs, contentClassName, className } = props
+  const { tabs, contentStyle, tabsStyle } = props
   const tabPanel = tabs[tabIndex]
   const Component = tabPanel?.component
   const componentProps = tabPanel?.props
 
-  const handleChange = (event, newTabIndex) => {
+  const handleChange = (_event, newTabIndex) => {
     setTabIndex(newTabIndex)
   }
 
+  const StyledTabs = styled(Tabs)<TabsProps>({
+    height: 24,
+    minHeight: 24,
+
+    '& .MuiTabs-indicator': {
+      display: 'none'
+    },
+    ...tabsStyle
+  })
+
+  const StyledTab = styled(Tab)<TabProps>(({ theme }) => ({
+    minHeight: 24,
+    minWidth: 60,
+    height: 1,
+    width: 65,
+    padding: 0,
+    fontSize: 10,
+    cursor: 'pointer',
+    textAlign: 'center',
+    backgroundColor: theme.colors.tab.regular,
+    color: theme.colors.text.black,
+
+    '&:hover': {
+      backgroundColor: theme.colors.tab.hover,
+      color: theme.colors.text.white
+    },
+
+    '&.Mui-selected': {
+      color: theme.colors.text.white,
+      backgroundColor: theme.colors.tab.active
+    },
+
+    ...tabsStyle
+  }))
+
+  const TabListContent = styled('div')({
+    ...contentStyle
```

Comment:
Instead of passing `contentStyle` and `tabsStyle` as props from parent, let's write styles right here, next to component itself.

Reply:
Here's an example. We can even use the same scrollbar mixin, :

```tsx
  const TabListContent = styled('div')(({ theme }) => ({
    ...scrollbarThin(theme),

    display: 'flex',
    flexDirection: 'row',
    flexWrap: 'wrap',
    justifyContent: 'flex-start',
    overflowY: 'auto',
    width: '260px',

    '& > *': {
      margin: '25px 8px'
    }
  }))
```

Or better yet, use template string:

```tsx
  const TabListContent = styled('div')`
    display: flex;
    flex-direction: row;
    flex-wrap: wrap;
    justify-content: flex-start;
    overflow-y: auto;
    width: 260px;

    & > * {
      margin: 25px 8px;
    }
    ${(props) => scrollbarThin(props.theme)}
  `
```
I suggest that we try to use the latter approach with template literals with other components here too.